README.md

```diff
@@ -1,2 +1,42 @@
-# odh-manifests
-A repository for Open Data Hub Kustomize manifests extending upstream Kubeflow manifests
+# Open Data Hub Manifests
+A repository for [Open Data Hub](https://opendatahub.io) components Kustomize manifests.
+
+## Community
+
+* Website: https://opendatahub.io
+* Documentation: https://opendatahub.io/docs.html (applicable for version 0.5.1)
+* Mailing lists: https://opendatahub.io/community.html
+* Community meetings: https://gitlab.com/opendatahub/opendatahub-community
+
+## Components
+
+Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers. The components currently available are:
+
+* [JupyterHub](jupyterhub/README.md)
+* Spark Operator
+* Kafka
+* Superset
+* AI Library
+* Grafana & Prometheus
+
+Some components are still in process of conversion from Ansible Operator based version on [Gitlab](https://gitlab.com/opendatahub/opendatahub-operator/)
+
+* Argo
+* Seldon
+* Data Catalog
+    * Hue
+    * Hive
+    * Thrift Server
+
+There are also other components planned to be added
+
+* Airflow
+
+## Deploy
+
+We are relying on [Kustomize](https://github.com/kubernetes-sigs/kustomize), [kfctl](https://github.com/kubeflow/kfctl) and [Kubeflow Operator](https://github.com/kubeflow/kfctl/blob/master/operator.md) for deployment.
+
+The two ways how to deploy at the moment are
```

Comment:
probably don't need the word "how" here.   Also, "at the moment" is possibly also unnecessary.

"The two ways to deploy are:"